platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -299,12 +299,28 @@ public Date getExpires(Subject subject) {
      */
     public SecurityManager getSecurityManager() {
         BundleContext context = getBundleContext();
+
         if (context != null) {
             ServiceReference securityManagerRef =
                     context.getServiceReference(SecurityManager.class);
-            return (SecurityManager) context.getService(securityManagerRef);
+
+            int tries = 0;
+            while (tries < 10) {
+                try {
+                    SecurityManager securityManager = (SecurityManager) context.getService(
+                            securityManagerRef);
+                    return securityManager;
+                } catch (NullPointerException e) {
```

Comment:
❓ Anything we can check to prevent the `NullPointerException`? Catching it seems a little odd and may not work in the future.